assets/eip-5007/contracts/IERC5007.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: CC0
+
+pragma solidity ^0.8.0;
+
+interface IERC5007 /* is IERC721 */ {
+    /// @notice Get the start time of the NFT
+    /// @dev Throws if `tokenId` is not valid NFT
+    /// @param tokenId  The tokenId of the NFT
+    /// @return The start time of the NFT
+    function startTime(uint256 tokenId) external view returns (uint64);
+    
+    /// @notice Get the end time of the NFT
+    /// @dev Throws if `tokenId` is not valid NFT
+    /// @param tokenId  The tokenId of the NFT
+    /// @return The end time of the NFT
+    function endTime(uint256 tokenId) external view returns (uint64);
+
+    /// @notice Check the NFT is valid now 
+    /// @dev Throws if `tokenId` is not valid NFT
+    /// @param tokenId  The tokenId of the NFT
+    /// @return The the NFT is valid now
+    /// if(startTime <= now <= endTime) {return true;} else {return false;}
```

Comment:
> `isCurrent` is more clear than `isValidNow`, since the term _valid_ has more generic implications about the property of the NFT.

Thanks.  Different project may has different definition of 'valid', so I  removed `isValidNow` function.